spec/factories/vacancies.rb

```diff
@@ -10,42 +10,43 @@
       create_list :document, 3, vacancy: vacancy
     end
 
-    job_title { Faker::Lorem.sentence[1...30].strip }
-    job_description { Faker::Lorem.paragraph(sentence_count: 4) }
+    application_link { Faker::Internet.url }
```

Comment:
Is alphabetical more useful than in order of hiring staff journey? I'm not sure, but agree we should have an approach!

Reply:
I think our approach is alphabetization for long lists